examples/test/gltf-model-error/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>glTF Model</title>
+    <meta name="description" content="glTF Model - A-Frame">
```

Comment:
I know it's misleading the test examples are not actually tests but simple examples that test very specific features. For this we should probably write a [unit test](https://github.com/aframevr/aframe/tree/master/tests)

Reply:
OK - I will have a go at a Unit Test.

Reply:
Unit Tests now added.

Reply:
Can we leave this in as well, or do you prefer me to delete it?

Reply:
I would remove this so `examples/test` is consistent. It serves the purpose to test visually specific features. Your unit test handles the failure case.

Reply:
OK - removed it.